values.md

```diff
@@ -2,7 +2,7 @@
 title: "Kubernetes Community Values"
 weight: 2
 description: |
-  This page indexes all of the mentoring initiatives for contributing to the Kubernetes project. For end user mentoring initiatives, check out KubeCon and other CNCF events and programs.
+  This page indexes all of the community values that are part of the Kubernetes Community culture. The indexed values have evolved over time, pushing our project and peers toward constant improvement.
```

Comment:
```suggestion
  The Kubernetes Community values are the keystone to the ongoing success of the project. These principles guide every aspect of the Kubernetes project.
```

This suggestion mirrors the values description we used on https://k8s.io/community page 👍